cost/azure/savings_realized/azure_savings_realized.pt

```diff
@@ -7,7 +7,7 @@ category "Cost"
 severity "low"
 default_frequency "monthly"
 info(
-  version: "3.7",
+  version: "3.8",
   provider: "Azure",
   service: "Compute",
   policy_set: "Savings Realized"
```

Comment:
Regarding Line 194 - I know you did not make any changes to this as part of this Pull Request, however:

the comment on Line 194 currently says `# Gather list of bill connections from Optima data`

Could you replace the word `Optima` with `Flexera CCO`?